bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gnirs/InstGNIRS.java

```diff
@@ -155,6 +161,7 @@ private static PropertyDescriptor initProp(String propName, boolean query, boole
     private Decker _decker = Decker.DEFAULT;
     private Filter _filter = Filter.DEFAULT;
     private WellDepth _wellDepth = WellDepth.DEFAULT;
+    private HartmannMask _hartmannMask = HartmannMask.DEFAULT;
```

Comment:
I believe this property along with `Focus` should be included in `getParamSet` and `setParamSet` as well.

Reply:
It wasn't clear why some properties are included but others are not.  I copied the focus bits from NIRI where they are not included.  :-\